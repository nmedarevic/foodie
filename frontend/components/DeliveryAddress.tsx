import { DeliveryForm } from "./DeliveryForm";
import { Section } from "./Common/Section";

export const DeliveryAddress = ({onChange}: {onChange: Function}) => (
  <Section>
    <div className='flex flex-col'>
      <h2 className='text-2xl'>Add delivery address</h2>
      <DeliveryForm onChange={onChange} />
    </div>
  </Section>
)
