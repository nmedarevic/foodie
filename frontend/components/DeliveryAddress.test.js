import React from 'react'
import renderer from 'react-test-renderer'
import {DeliveryAddress} from './DeliveryAddress'

it('Renders DeliveryAddress without props', () => {
  const tree = renderer.create(<DeliveryAddress />).toJSON()
  expect(tree).toMatchSnapshot()
})
