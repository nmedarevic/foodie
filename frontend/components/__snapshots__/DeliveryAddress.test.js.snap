// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Delivery day with a selected day 1`] = `
<section
  className="bg-white py-8 flex"
>
  <div
    className="container mx-auto flex items-center flex-wrap flex-col pt-4 pb-12 self-center"
  >
    <div
      className="flex flex-col"
    >
      <h1
        className="text-2xl"
      >
        Select first delivery day
      </h1>
      <div
        className="flex"
      >
        <div
          className="mt-5 mr-5 cursor-pointer nline-flex items-center px-6 py-3 text-black font-semibold shadow-sm bg-gray-200"
          data-test="one"
          onClick={[Function]}
        >
          one
        </div>
        <div
          className="mt-5 mr-5 cursor-pointer nline-flex items-center px-6 py-3 text-white font-semibold shadow-sm bg-blue-500"
          data-test="two"
          onClick={[Function]}
        >
          two
        </div>
      </div>
    </div>
  </div>
</section>
`;

exports[`Renders Delivery day with basic props 1`] = `
<section
  className="bg-white py-8 flex"
>
  <div
    className="container mx-auto flex items-center flex-wrap flex-col pt-4 pb-12 self-center"
  >
    <div
      className="flex flex-col"
    >
      <h1
        className="text-2xl"
      >
        Select first delivery day
      </h1>
      <div
        className="flex"
      >
        <div
          className="mt-5 mr-5 cursor-pointer nline-flex items-center px-6 py-3 text-white font-semibold shadow-sm bg-blue-500"
          data-test="one"
          onClick={[Function]}
        >
          one
        </div>
        <div
          className="mt-5 mr-5 cursor-pointer nline-flex items-center px-6 py-3 text-black font-semibold shadow-sm bg-gray-200"
          data-test="two"
          onClick={[Function]}
        >
          two
        </div>
      </div>
    </div>
  </div>
</section>
`;

exports[`Renders Delivery day without props 1`] = `
<section
  className="bg-white py-8 flex"
>
  <div
    className="container mx-auto flex items-center flex-wrap flex-col pt-4 pb-12 self-center"
  >
    <div
      className="flex flex-col"
    >
      <h1
        className="text-2xl"
      >
        Select first delivery day
      </h1>
      <div
        className="flex"
      />
    </div>
  </div>
</section>
`;

exports[`Renders DeliveryAddress without props 1`] = `
<section
  className="bg-white py-8 flex"
>
  <div
    className="container mx-auto flex items-center flex-wrap flex-col pt-4 pb-12 self-center"
  >
    <div
      className="flex flex-col"
    >
      <h2
        className="text-2xl"
      >
        Add delivery address
      </h2>
      <div
        className="flex flex-col mt-2 mb-2"
      >
        <label
          htmlFor="firstName"
        >
          First Name
        </label>
        <input
          className="shadow-sm"
          id="firstName"
          name="firstName"
          onChange={[Function]}
          type="text"
        />
      </div>
      <div
        className="flex flex-col mt-2 mb-2"
      >
        <label
          htmlFor="lastName"
        >
          Last Name
        </label>
        <input
          className="shadow-sm"
          id="lastName"
          name="lastName"
          onChange={[Function]}
          type="text"
        />
      </div>
      <div
        className="flex flex-col mt-2 mb-2"
      >
        <label
          htmlFor="email"
        >
          Email
        </label>
        <input
          className="shadow-sm"
          id="email"
          name="email"
          onChange={[Function]}
          type="text"
        />
      </div>
      <div
        className="flex flex-col mt-2 mb-2"
      >
        <label
          htmlFor="address"
        >
          Address
        </label>
        <input
          className="shadow-sm"
          id="address"
          name="address"
          onChange={[Function]}
          type="text"
        />
      </div>
      <div
        className="flex flex-col mt-2 mb-2"
      >
        <label
          htmlFor="postal"
        >
          Postal code
        </label>
        <input
          className="shadow-sm"
          id="postal"
          name="postal"
          onChange={[Function]}
          type="number"
        />
      </div>
    </div>
  </div>
</section>
`;
